technical_analysis_automation/head_shoulders.py

```diff
@@ -21,82 +22,92 @@ class HSPattern:
     head: int = -1
 
     # Price of the parts of the H&S pattern. _p stands for price.
-    l_shoulder_p: float = -1
-    r_shoulder_p: float = -1
-    l_armpit_p: float = -1
-    r_armpit_p: float = -1
-    head_p: float = -1
+    l_shoulder_price: float = -1.0
+    r_shoulder_price: float = -1.0
+    l_armpit_price: float = -1.0
+    r_armpit_price: float = -1.0
+    head_price: float = -1.0
```

Comment:
nice, yeah the original names are triggering me. am focusing on the rolling window calc for now